README.md

```diff
@@ -1,2 +1,89 @@
 # joi-image-extension
-Joi extension for validating image size and type
+
+A [Joi](https://github.com/hapijs/joi) extension for validating image sizes and types.
+
+## Requirements
+
+* [Node.js](https://nodejs.org/en/) 6.x+
+* [Joi](https://github.com/hapijs/joi) 10.x+
+
+## Installation
+
+```sh
+npm install joi-image-extension --save
+```
+
+## Usage
+
+```js
+const fs = require('fs')
+const BaseJoi = require('joi')
+const ImageExtension = require('joi-image-extension')
+const concat = require('concat-stream')
+
+const Joi = BaseJoi.extend(ImageExtension)
+
+const schema = Joi
+  .image()
+  .minDimensions(100, 50)
+
+fs
+  .createReadStream('./avatar.jpg')
+  .pipe(concat(image => {
+    Joi.validate(image, schema, (err, value) => {
+      // Handle validation result...
+    })
+  }))
+
+```
+
+## API
+
+### `image.minWidth(minimum pixel width)`
```

Comment:
When looking at the README as rendered by GitHub rather than as Markdown, this formatting looks a bit odd as the backtick usage implies a function signature. Maybe camelcase or just shorten them since the description immediately after explains it, `image.minWidth(minimumPixelWidth)`/`image.minWidth(minimum)`?

Reply:
@randallagordon I agree, it's kinda weird, but I'm just following suit with how Joi documents its stuff, which seems to be `(namespace).(function)`, e.g.: https://github.com/hapijs/joi/blob/v13.1.0/API.md#anyallowvalue

Reply:
They don't break them up with spaces such that it is displaying invalid code, however:

![image](https://user-images.githubusercontent.com/635388/34749410-21086500-f556-11e7-9cdb-f139fe15f48f.png)

vs:

![image](https://user-images.githubusercontent.com/635388/34749404-167a6f0c-f556-11e7-8b98-924e39a5ee42.png)